components/header.tsx

```diff
@@ -17,6 +17,7 @@ const HeaderlessRoutes = [
     '/orders/[orderId]',
     '/orders/[orderId]/labels',
     '/orders/[orderId]/modal',
+    '/product/[productId]',
```

Comment:
🍹 Maybe this has already been discussed, but this route name is kind of ambiguous. Could we come up with something a little more descriptive to differentiate this route from `/products/[pid]`?

Reply:
Oh yeah I spoke with Chaz about the route name and we went this one, the change from pid to productId was to differentiate it, front the other route.

Reply:
yeah, the more I think about this, the more I think maybe we should change this a bit to be more descriptive/ clear - maybe `productAppExtension`